doc/deployment/amazon_web_services.md

```diff
@@ -216,3 +216,8 @@ pachctl             1.4.6
 pachd               1.4.6
 ```
 
+## Production Deployment
+
+For production deployments we recommend setting up AWS CloudFront. This is helpful because of S3 rate limits that AWS puts in place that can limit throughput for your cluster reading data.
```

Comment:
Just some grammar stuff:

"...up AWS CloudFront.  AWS puts S3 rate limits in place that can limit the data throughput for your cluster, and CloudFront helps mitigate this issue."